composer.json

```diff
@@ -4,46 +4,58 @@
     "type": "project",
     "description": "Symfony RAD API Edition for powerful REST applications.",
     "autoload": {
-        "psr-4": { "": "src/" }
+        "psr-4": {
+            "AppBundle\\": "src/AppBundle",
+            "DemoBundle\\": "src/DemoBundle"
+        },
+        "classmap": ["app/AppKernel.php", "app/AppCache.php"]
     },
     "require": {
         "php": "^5.6|^7.0",
 
-        "sylius/resource-bundle":               "0.15",
+        "sylius/resource-bundle": "dev-master",
         "incenteev/composer-parameter-handler": "~2.0",
-        "sensio/distribution-bundle":           "2.3.*",
-        "friendsofsymfony/rest-bundle":         "1.5.*@dev",
-        "sensio/generator-bundle":              "~2.3",
-        "symfony/symfony":                      "~2.3",
-        "symfony/monolog-bundle":               "~2.3",
-        "doctrine/orm":                         "~2.3"
+        "friendsofsymfony/rest-bundle": "^1.8",
+        "symfony/symfony": "~3.2",
+        "symfony/monolog-bundle": "~2.3",
+        "doctrine/orm": "~2.3"
```

Comment:
Can we sort dependencies alphabetically? Also `^2.5` is a minimum for Symfony 3+.